system-test/data/zonefile.zone

```diff
@@ -1,2 +1,8 @@
 {DNS_DOMAIN} 21600 IN SPF "v=spf1" "mx:{DNS_DOMAIN}" "-all"
 {DNS_DOMAIN} 21600 IN TXT "google-site-verification=xxxxxxxxxxxxYYYYYYXXX"
+{DNS_DOMAIN} 21600 MX 10 mail.example.com.
+{DNS_DOMAIN} 21600 MX 30 mail.example.com.
+{DNS_DOMAIN} A 128.9.0.32
+             A 10.1.0.52
+{DNS_DOMAIN} A 10.2.0.27
+             A 128.9.0.33
```

Comment:
let's add new line at the end of file :)

Reply:
Done!